frontend/src/_ui/Select/SelectComponent.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import _ from 'lodash';
+import Select from 'react-select';
+import defaultStyles from './styles';
+
+export const SelectComponent = ({ options = [], value, onChange, ...restProps }) => {
+  const darkMode = localStorage.getItem('darkMode') === 'true';
+  const { styles, hasSearch = false, height, width, placeholder = 'Select..' } = restProps;
+
+  const useStyles = styles || defaultStyles(darkMode, width, height);
```

Comment:
```suggestion
  const useStyles = !_.isEmpty(styles) ? styles : defaultStyles(darkMode, width, height);
```

cos `{} || ...` would return `{}` & `[] || ...` would return `[]` would miss out on default styles if styles are passed as one of those